### ISSUE TITLE
[Game] Single sample downloader

### ISSUE THREAD

**Issue by zoeyfan** [flutter/flutter#149282]

**Comment by parlough** [comment]
@filiph Was https://github.com/flutter/games/pull/59 to resolve this issue? Can this be closed as done?

**Comment by sfshaza2** [comment]
Hmmm. Does this affect docs.flutter.dev?

**Comment by filiph** [comment]
I don't think so. If I remember correctly, I looked at docs.flutter.dev to see if there are any obvious places where to put a mention of the downloader, but didn't find any. I think it's better to send people to the github repo and then let them choose there if they want to use the downloader, just look at the code on GitHub, or clone the whole repo.
